### ISSUE TITLE
add json `list` output

### ISSUE THREAD

**Issue by noahcoad** [jakehilborn/displayplacer#141]
Would be helpful to get `displayplacer list` output in json format to be able to more easily programmatically retrieve display parameters, such as id, current mode, resolution, etc

**Comment by maximejobin** [comment]
I totally agree. I'm making a script and the values also contain comments so it is not fun to manipulate.

For instance:

    Origin: (0,0) - main display
    Rotation: 0 - rotate internal screen example (may crash computer, but will be rotated after rebooting): `displayplacer "id:37D8832A-2D66-02CA-B9F7-8F30A301B230 degree:90"`

Should be:

    Origin: (0,0)
    Rotation: 0

Samething for:

     mode 2: res:1024x768 hz:60 color_depth:4 scaling:on <-- current mode

That should be:

     mode 2: res:1024x768 hz:60 color_depth:4 scaling:on

... with probably a `mode` value.

To make things easier for developers, please consider a JSON output.

**Comment by krin-san** [comment]
Was also interested if `displayplacer` can output JSON. Meanwhile in order to turn off external displays I connect to just to charge my laptop I had to do the following:
```
displayplacer list | tail -n 1 | grep -o 'id:[^ ]*' | tail -n 1 | xargs -I{} displayplacer "{} enabled:false"
```

**Comment by bartmeuris** [comment]
Submitted PR #143 for this functionality. Feedback is welcome.

**Comment by milichev** [comment]
@noahcoad  @krin-san  @maximejobin , you can use [this script](https://gist.github.com/milichev/3de6d332ee056374858d9a4377fa5053) while the [PR](https://github.com/jakehilborn/displayplacer/pull/143) is not pushed.
It parses the output of `displayplacer list` and substitutes the actual values.
Currently supports only one external display.

**Comment by bartmeuris** [comment]
FYI: Since creating the PR, I've switched to Hammerspoon using the `hs.screen` module for my display scripting/logic, so you don't need json parsing anymore and have access to data structures directly in a Lua script.

**Comment by noahcoad** [comment]
Thanks @milichev for putting up #143 — really appreciate the work on this!
